modin/core/dataframe/pandas/partitioning/partition_manager.py

```diff
@@ -712,7 +712,7 @@ def update_bar(pbar, f):
 
         num_splits = NPartitions.get()
         put_func = cls._partition_class.put
-        row_chunksize, col_chunksize = compute_chunksize(df, num_splits)
+        row_chunksize, col_chunksize = compute_chunksize(*df.shape, num_splits)
```

Comment:
Since `compute_chunksize` parameters are keyword arguments I would prefer to pass them explicitly.

Reply:
Why? Current version looks shorter and more intuitive.

Reply:
Someone may change parameter order and then we would get incorrect flow.

Reply:
Arguments can be passed to any function in `keyword` form, instead of `positional` form. We don't do that everywhere, do we?

Reply:
Yes, we do not do that but I would prefer. If the function had positional arguments only, we could pass the arguments as it is now.